runtime/src/chpl-init.c

```diff
@@ -141,6 +141,7 @@ void chpl_rt_init(int argc, char* argv[]) {
   // UTF-8 functions (e.g. wcrtomb) work as
   // indicated by the locale environment variables.
   setlocale(LC_CTYPE,"");
+  qio_set_glocale();
```

Comment:
I think this used to be here but we moved it into qio functions e.g. qio_channel_scan_string for some reason I can't remember. Anyway I personally think it makes more sense here.

I'm fine with it being here but if it is you should remove the other calls to it.